backend/scripts/set_src_dest_envs.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+export SRC_ENV=${1:-prod}
+export DEST_ENV=${2:-dev}
+
+if [[ ! "$SRC_ENV" =~ ^(dev|staging|prod)$ ]]; then
```

Comment:
given line 16... is there any situation where we'd want the src env to be dev?

Reply:
Only for testing, so basically no.  I thought about just enumerating the possible allowed combinations to make it clearer. Maybe I'll do that.

Reply:
I think just removing dev as a src option would work?

Reply:
I thought there might be times when we wanted to mirror from staging to dev. E.g., someone finds a bug in staging and we want to verify it's fixed in dev, and that requires having the same data as in staging. The scenario is a bit of a stretch, but I didn't want to rule it out.

Basically, wanted to allow any src/dest pair that wasn't "upward".

Reply:
wait, wouldnt staging be the src and dev the destination then? so we still wouldnt want to dev as the src?